docker/Dockerfile.cypress

```diff
@@ -9,11 +9,11 @@ COPY ./package.json .
 
 RUN yarn add cypress-circleci-reporter
 
-COPY ./cypress.json .
+COPY ./cypress.json ./cypress.json
```

Comment:
is this necessary or was this a change made while working a solution?

I'm pretty sure that

```docker
COPY ./cypress.json .
```

is identical to
```docker
COPY ./cypress.json ./cypress.json
```
so it probably doesn't make a huge deal.

Reply:
@hoorayimhelping the change was made when I was trouble shooting the cypress dockerfile. All the other paths were listed twice, so I updated that one to match. If you want it removed, I can revert that line back.

Reply:
@gspletzer since this PR is passing and it's more consistent and explicit, I think it's fine to leave it.